ic-os/guestos/defs.bzl

```diff
@@ -28,28 +28,40 @@ def image_deps(mode, malicious = False):
+            "//publish/binaries:ic-btc-adapter": "/opt/ic/bin/ic-btc-adapter:0755",         # Required by the Bitcoin integration.
+            "//rs/https_outcalls/adapter:ic-https-outcalls-adapter": "/opt/ic/bin/ic-https-outcalls-adapter:0755",  # Required by the HTTPS outcalls feature. `//publish/binaries:ic-https-outcalls-adapter` is for testing and must NOT be used here
+            "//publish/binaries:ic-crypto-csp": "/opt/ic/bin/ic-crypto-csp:0755",           # Crypto provider required by the IC protocol.
+            "//publish/binaries:orchestrator": "/opt/ic/bin/orchestrator:0755",             # Replica process manager. Required for upgrades, node addition, etc.
+            ("//publish/malicious:replica" if malicious else "//publish/binaries:replica"): "/opt/ic/bin/replica:0755",  # Main protocol binary. Installs the malicious replica if set.
+            "//publish/binaries:ic-boundary-tls": "/opt/ic/bin/ic-boundary:0755",           # API Boundary Node. The same GuestOS is used both for the replica and API boundary nodes.
+            # TODO consensus
             "//publish/binaries:ic-consensus-pool-util": "/opt/ic/bin/ic-consensus-pool-util:0755",
-            "//rs/https_outcalls/adapter:ic-https-outcalls-adapter": "/opt/ic/bin/ic-https-outcalls-adapter:0755",  # `//publish/binaries:ic-https-outcalls-adapter` is for testing and must NOT be used here
-            "//publish/binaries:ic-crypto-csp": "/opt/ic/bin/ic-crypto-csp:0755",
+            # TODO consensus
             "//publish/binaries:ic-regedit": "/opt/ic/bin/ic-regedit:0755",
+            # TODO consensus
             "//publish/binaries:ic-recovery": "/opt/ic/bin/ic-recovery:0755",
-            "//publish/binaries:orchestrator": "/opt/ic/bin/orchestrator:0755",
-            "//publish/binaries:ic-boundary-tls": "/opt/ic/bin/ic-boundary:0755",
-            ("//publish/malicious:replica" if malicious else "//publish/binaries:replica"): "/opt/ic/bin/replica:0755",  # Install the malicious replica if set
+            # TODO crypto, node
+            "//publish/binaries:fstrim_tool": "/opt/ic/bin/fstrim_tool:0755",
+            # TODO node
+            "//publish/binaries:guestos_tool": "/opt/ic/bin/guestos_tool:0755",
+            # TODO dre
             "//publish/binaries:metrics-proxy": "/opt/ic/bin/metrics-proxy:0755",
+            # TODO node
             "//publish/binaries:nft-exporter": "/opt/ic/bin/nft-exporter:0755",
-            "//publish/binaries:sandbox_launcher": "/opt/ic/bin/sandbox_launcher:0755",
+            # TODO messaging
             "//publish/binaries:state-tool": "/opt/ic/bin/state-tool:0755",
+            # TODO node
             "//publish/binaries:vsock_guest": "/opt/ic/bin/vsock_guest:0755",
+            # TODO node
             "//cpp:infogetty": "/opt/ic/bin/infogetty:0755",
+            # TODO node
             "//cpp:prestorecon": "/opt/ic/bin/prestorecon:0755",
 
             # additional libraries to install
+            # TODO dre
```

Comment:
Allows referring to the guest IPv6 by name `guestos` from host, and host as `hostos` from guest.